solidity/depool/README.md

```diff
@@ -1089,3 +1257,73 @@ This can be caused by DePool setup errors, low balance on some of the contracts
     ```bash
     bash -x ./validator_depool.sh
     ```
+    
+## 4. DePool function terminates with error in TONOS-CLI
+
+In TONOS-CLI output it looks like this:
+
+```bash
+message: Contract execution was terminated with error
+message_processing_state: null
+data: {
+  "account_address": "0:3187b4d738d69776948ca8543cb7d250c042d7aad1e0aa244d247531590b9147",
+  "config_server": "net.ton.dev",
+  "exit_code": 140,
+  "function_name": "constructor",
+  "original_error": {
+    "code": 1006,
+    "core_version": "0.27.0",
+    "data": {
+      "block_id": "20c8932bc6c7baa90c2d099595b432607355b79d95293061dc6feec392675975",
+      "block_time": "Thu, 19 Nov 2020 13:24:37 +0300 (1605781477)",
+      "expiration_time": "Thu, 19 Nov 2020 13:24:35 +0300 (1605781475)",
+      "message_id": "eae70ebcf84fd9ebcffc544908ca637d9a46a8bdffdab24db6749d003608969e",
+      "sending_time": "Thu, 19 Nov 2020 13:23:35 +0300 (1605781415)"
+    },
+    "message": "Message was not delivered within the specified timeout",
+    "message_processing_state": null,
+    "source": "node"
+  },
+```
+
+The error code is indicated in the `"exit_code"` line.
+
+Possible error codes and their meanings:
+
+`101` - message sender is not owner (message public key is wrong). Please check your key file or seed phrase.
+
+`108` - function cannot be called by external message.
+
+`113` - message sender is not in validator pool.
+
+`114` - DePool is closed.
+
+`116` - participant with such address does not exist.
+
+`128` - message sender is not owner (message public key is wrong) or self call.
+
+`129` - incorrectly defined stake parameters during DePool deployment (`minStake` < 1 token or `minStake` > `validatorAssurance`). Please check your parameters in deploy message.
+
+`130` - DePool deployment isn't signed with public key. Please check your key file or seed phrase.
+
+`133` - validator address isn't defined in standard form during DePool deployment. Please check your parameters in deploy message.
```

Comment:
validator address passed to constructor is not of add_std type

Reply:
corrected